packages/chain-bridge/src/adapters/LayerZeroAdapter.sol

```diff
@@ -215,12 +354,14 @@ contract LayerZeroAdapter is
         // Cache LayerZero EID to avoid redundant storage reads
         uint32 lzDstEid = _getLayerZeroEid(params.destinationChainId);
 
-        // Validate fee requirements using helper
-        LayerZeroMessagingHelper.validateFeeRequirements(options, msg.value);
+        // Validate fee requirements
+        if (options.msgValue > 0 && msg.value < options.msgValue) {
```

Comment:
`options.msgValue` is not native fee - it's the value we send to the target chain